src/host/dns.h

```diff
@@ -56,7 +56,11 @@ namespace asynchost
              service.c_str(),
              &hints)) < 0)
         {
-          LOG_FAIL_FMT("uv_getaddrinfo failed: {}", uv_strerror(rc));
+          LOG_FAIL_FMT(
```

Comment:
Should we add the same info for the async version a few lines up?

Reply:
yes thanks